probcal/experiments/mixup_test.py

```diff
@@ -0,0 +1,50 @@
+import torch
+import matplotlib.pyplot as plt
+from probcal.enums import DatasetType, ImageDatasetName
+from probcal.utils.experiment_utils import get_model, get_datamodule
+
+from probcal.transforms import MixUpTransform
+
+L = 0.25
```

Comment:
In some places we `import lightning as L`, so just in case weird errors pop up, we should rename this variable

Reply:
This script was just to test out the MixUpTransform and is no longer needed. 

Deleted.